package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "tupaia",
+  "private": true,
```

Comment:
This is ok, I think the `private` refers to npm rather than github/open-source

Reply:
Exactly!